src/platform/unittest/DataPathTest.cpp

```diff
@@ -535,8 +533,8 @@ struct CxPlatSocket {
         if (QUIC_SUCCEEDED(InitStatus)) {
             CxPlatSocketGetLocalAddress(Socket, &Route.LocalAddress);
             CxPlatSocketGetRemoteAddress(Socket, &Route.RemoteAddress);
-#ifdef QUIC_USE_RAW_DATAPATH
-            if (!QuicAddrIsWildCard(&Route.RemoteAddress)) {
+            if (!!(CxPlatDataPathGetSupportedFeatures(Datapath) & CXPLAT_DATAPATH_FEATURE_RAW_SOCKET) &&
```

Comment:
To make this a bit cleaner (IMO), let's add a new helper function to `CxPlatDataPath` called `IsSupported(flag)` so you can change this line to:
```suggestion
            if (Datapath.IsSupported(CXPLAT_DATAPATH_FEATURE_RAW_SOCKET) &&
```
And then use it below too.